<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddEventDialog</class>
 <widget class="QDialog" name="AddEventDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>353</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Додати подію</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
background: #FFFFFF;
}</string>
  </property>
  <widget class="QPushButton" name="btnOK">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>440</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
     color: #ffffff;
     font-weight: bold;
     border: 1px solid #8f8f91;
     border-radius: 6px;
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #ffebb0, stop: 1  #ff911c);
     min-width: 80px;
 }

 QPushButton:pressed {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0  #ff911c, stop: 1 #ffebb0);
 }

 QPushButton:flat {
     border: none; /* для плоской кнопки границы нет */
 }

 QPushButton:default {
     border-color: navy; /* делаем кнопку по умолчанию выпуклой */
 }


</string>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>470</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
     color: #ffffff;
     font-weight: bold;
     border: 1px solid #8f8f91;
     border-radius: 6px;
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #ffebb0, stop: 1   #ff911c);
     min-width: 80px;
 }

 QPushButton:pressed {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0  #ff911c, stop: 1 #ffebb0);
 }

 QPushButton:flat {
     border: none; /* для плоской кнопки границы нет */
 }

 QPushButton:default {
     border-color: navy; /* делаем кнопку по умолчанию выпуклой */
 }


</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Виберіть тип події:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="teDescription">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>271</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit:enabled{
border: 1px solid #ff911c;
 border-radius:10px;
 background-color: palette(base);
}
QTextEdit:disabled{
 border-radius:10px;
border: 1px solid #CCCCCC; 
background-color: palette(base);
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Опис події: </string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>271</width>
     <height>42</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QRadioButton" name="rbBusiness">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QRadioButton::indicator {
    width: 13px;
    height: 13px; 
}

QRadioButton::indicator::unchecked {
    image: url(:/rc/radioButtonUncheckedIcon);
}

QRadioButton::indicator:unchecked:hover {
     image: url(:/rc/radioButtonUncheckedIcon);
}

QRadioButton::indicator:unchecked:pressed {
    image: url(:/rc/radioButtonCheckedIcon);
}

QRadioButton::indicator::checked {
    image: url(:/rc/radioButtonCheckedIcon);
}

QRadioButton::indicator:checked:hover {
    image: url(:/rc/radioButtonCheckedIcon);
}

QRadioButton::indicator:checked:pressed {
   image: url(:/rc/radioButtonUncheckedIcon);
}</string>
      </property>
      <property name="text">
       <string>Заняття</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QRadioButton" name="rbTask">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QRadioButton::indicator {
    width: 13px;
    height: 13px; 
}

QRadioButton::indicator::unchecked {
    image: url(:/rc/radioButtonUncheckedIcon);
}

QRadioButton::indicator:unchecked:hover {
     image: url(:/rc/radioButtonUncheckedIcon);
}

QRadioButton::indicator:unchecked:pressed {
    image: url(:/rc/radioButtonCheckedIcon);
}

QRadioButton::indicator::checked {
    image: url(:/rc/radioButtonCheckedIcon);
}

QRadioButton::indicator:checked:hover {
    image: url(:/rc/radioButtonCheckedIcon);
}

QRadioButton::indicator:checked:pressed {
   image: url(:/rc/radioButtonUncheckedIcon);
}</string>
      </property>
      <property name="text">
       <string>Задача</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QRadioButton" name="rbMeet">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QRadioButton::indicator {
    width: 13px;
    height: 13px; 
}

QRadioButton::indicator::unchecked {
    image: url(:/rc/radioButtonUncheckedIcon);
}

QRadioButton::indicator:unchecked:hover {
     image: url(:/rc/radioButtonUncheckedIcon);
}

QRadioButton::indicator:unchecked:pressed {
    image: url(:/rc/radioButtonCheckedIcon);
}

QRadioButton::indicator::checked {
    image: url(:/rc/radioButtonCheckedIcon);
}

QRadioButton::indicator:checked:hover {
    image: url(:/rc/radioButtonCheckedIcon);
}

QRadioButton::indicator:checked:pressed {
   image: url(:/rc/radioButtonUncheckedIcon);
}</string>
      </property>
      <property name="text">
       <string>Зустріч</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QRadioButton" name="rbBirthday">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QRadioButton::indicator {
    width: 13px;
    height: 13px; 
}

QRadioButton::indicator::unchecked {
    image: url(:/rc/radioButtonUncheckedIcon);
}

QRadioButton::indicator:unchecked:hover {
     image: url(:/rc/radioButtonUncheckedIcon);
}

QRadioButton::indicator:unchecked:pressed {
    image: url(:/rc/radioButtonCheckedIcon);
}

QRadioButton::indicator::checked {
    image: url(:/rc/radioButtonCheckedIcon);
}

QRadioButton::indicator:checked:hover {
    image: url(:/rc/radioButtonCheckedIcon);
}

QRadioButton::indicator:checked:pressed {
   image: url(:/rc/radioButtonUncheckedIcon);
}</string>
      </property>
      <property name="text">
       <string>День народження</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>42</x>
     <y>251</y>
     <width>271</width>
     <height>164</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Дата:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="dateEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>3</verstretch>
         </sizepolicy>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QDateEdit:enabled {
 border: 1px solid #ff911c;
 border-radius: 5px;
 background-color: white;
 }
QDateEdit:disabled {
 border: 1px solid #CCCCCC;
 border-radius: 5px;
 background-color: white;
 }

QDateEdit::up-arrow:enabled { 
border-left: 3px solid none;
border-right: 3px solid none;
 border-bottom: 3px solid black; 
width: 0px;
 height: 0px; 
}
QDateEdit::up-arrow:disabled { 
border-left: 3px solid none;
border-right: 3px solid none;
 border-bottom: 3px solid black; 
width: 0px;
 height: 0px; 
}


QDateEdit::up-button:enabled {
 width: 14px; 
height: 9px; 
background-color: #FF911C;
}
QDateEdit::up-button:disabled {
 width: 14px; 
height: 9px; 
background-color: #CCCCCC;
}

QDateEdit::up-button:hover {
 background-color: #FFD65E;
 border-top-right-radius: 5px; /* same radius as the QComboBox */
}
 
QDateEdit::down-arrow:enabled { 
border-left: 3px solid none;
border-right: 3px solid none; 
border-top: 3px solid black; 
width: 0px; 
height: 0px; 
}

QDateEdit::down-arrow:disabled { 
border-left: 3px solid none;
border-right: 3px solid none; 
border-top: 3px solid black; 
width: 0px; 
height: 0px; 
}


QDateEdit::down-button:enabled { 
width: 14px; 
height: 9px;
 background-color: #FF911C;
}
QDateEdit::down-button:disabled { 
width: 14px; 
height: 9px;
 background-color: #CCCCCC;
}
QDateEdit::down-button:hover {
 background-color: #FFD65E;
 border-bottom-right-radius: 3px;
}

QDateEdit::up-arrow:hover { 
border-left: 3px solid none;
border-right: 3px solid none;
 border-bottom: 3px solid black;
 width: 0px; 
height: 0px; 
}
QDateEdit::down-arrow:hover {
 border-left: 3px solid none;
border-right: 3px solid none;
 border-top: 3px solid black;
 width: 0px;
 height: 0px; 
}</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Початок:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTimeEdit" name="teStart">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QTimeEdit:enabled {
 border: 1px solid #ff911c;
 border-radius: 5px;
 background-color: white;
 }
QTimeEdit:disabled {
 border: 1px solid #CCCCCC;
 border-radius: 5px;
 background-color: white;
 }

QTimeEdit::up-arrow:enabled { 
border-left: 3px solid none;
border-right: 3px solid none;
 border-bottom: 3px solid black; 
width: 0px;
 height: 0px; 
}
QTimeEdit::up-arrow:disabled { 
border-left: 3px solid none;
border-right: 3px solid none;
 border-bottom: 3px solid black; 
width: 0px;
 height: 0px; 
}


QTimeEdit::up-button:enabled {
 width: 14px; 
height: 9px; 
background-color: #FF911C;
}
QTimeEdit::up-button:disabled {
 width: 14px; 
height: 9px; 
background-color: #CCCCCC;
}

QTimeEdit::up-button:hover {
 background-color: #FFD65E;
 border-top-right-radius: 5px; /* same radius as the QComboBox */

}
 
QTimeEdit::down-arrow:enabled { 
border-left: 3px solid none;
border-right: 3px solid none; 
border-top: 3px solid black; 
width: 0px; 
height: 0px; 
}

QTimeEdit::down-arrow:disabled { 
border-left: 3px solid none;
border-right: 3px solid none; 
border-top: 3px solid black; 
width: 0px; 
height: 0px; 
}


QTimeEdit::down-button:enabled { 
width: 14px; 
height: 9px;
 background-color: #FF911C;
}
QTimeEdit::down-button:disabled { 
width: 14px; 
height: 9px;
 background-color: #CCCCCC;
}
QTimeEdit::down-button:hover {
 background-color: #FFD65E;
 border-bottom-right-radius: 5px;
}

QTimeEdit::up-arrow:hover { 
border-left: 3px solid none;
border-right: 3px solid none;
 border-bottom: 3px solid black;
 width: 0px; 
height: 0px; 
}
QTimeEdit::down-arrow:hover {
 border-left: 3px solid none;
border-right: 3px solid none;
 border-top: 3px solid black;
 width: 0px;
 height: 0px; 
}</string>
        </property>
        <property name="locale">
         <locale language="Ukrainian" country="Ukraine"/>
        </property>
        <property name="currentSection">
         <enum>QDateTimeEdit::HourSection</enum>
        </property>
        <property name="displayFormat">
         <string>h:mm AP</string>
        </property>
        <property name="calendarPopup">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Кінець:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTimeEdit" name="teEnd">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QTimeEdit:enabled {
 border: 1px solid #ff911c;
 border-radius: 5px;
 background-color: white;
 }
QTimeEdit:disabled {
 border: 1px solid #CCCCCC;
 border-radius: 5px;
 background-color: white;
 }

QTimeEdit::up-arrow:enabled { 
border-left: 3px solid none;
border-right: 3px solid none;
 border-bottom: 3px solid black; 
width: 0px;
 height: 0px; 
}
QTimeEdit::up-arrow:disabled { 
border-left: 3px solid none;
border-right: 3px solid none;
 border-bottom: 3px solid black; 
width: 0px;
 height: 0px; 
}


QTimeEdit::up-button:enabled {
 width: 14px; 
height: 9px; 
background-color: #FF911C;
}
QTimeEdit::up-button:disabled {
 width: 14px; 
height: 9px; 
background-color: #CCCCCC;
}

QTimeEdit::up-button:hover {
 background-color: #FFD65E;
 border-top-right-radius: 5px; /* same radius as the QComboBox */

}
 
QTimeEdit::down-arrow:enabled { 
border-left: 3px solid none;
border-right: 3px solid none; 
border-top: 3px solid black; 
width: 0px; 
height: 0px; 
}

QTimeEdit::down-arrow:disabled { 
border-left: 3px solid none;
border-right: 3px solid none; 
border-top: 3px solid black; 
width: 0px; 
height: 0px; 
}


QTimeEdit::down-button:enabled { 
width: 14px; 
height: 9px;
 background-color: #FF911C;
}
QTimeEdit::down-button:disabled { 
width: 14px; 
height: 9px;
 background-color: #CCCCCC;
}
QTimeEdit::down-button:hover {
 background-color: #FFD65E;
 border-bottom-right-radius: 5px;
}

QTimeEdit::up-arrow:hover { 
border-left: 3px solid none;
border-right: 3px solid none;
 border-bottom: 3px solid black;
 width: 0px; 
height: 0px; 
}
QTimeEdit::down-arrow:hover {
 border-left: 3px solid none;
border-right: 3px solid none;
 border-top: 3px solid black;
 width: 0px;
 height: 0px; 
}</string>
        </property>
        <property name="locale">
         <locale language="Ukrainian" country="Ukraine"/>
        </property>
        <property name="currentSection">
         <enum>QDateTimeEdit::HourSection</enum>
        </property>
        <property name="displayFormat">
         <string>h:mm AP</string>
        </property>
        <property name="calendarPopup">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Час дедлайну:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateTimeEdit" name="deadlineDateTime">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QDateTimeEdit:enabled {
 border: 1px solid #ff911c;
 border-radius: 5px;
 background-color: white;
 }
QDateTimeEdit:disabled {
 border: 1px solid #CCCCCC;
 border-radius: 5px;
 background-color: white;
 }

QDateTimeEdit::up-arrow:enabled { 
border-left: 3px solid none;
border-right: 3px solid none;
 border-bottom: 3px solid black; 
width: 0px;
 height: 0px; 
}
QDateTimeEdit::up-arrow:disabled { 
border-left: 3px solid none;
border-right: 3px solid none;
 border-bottom: 3px solid black; 
width: 0px;
 height: 0px; 
}


QDateTimeEdit::up-button:enabled {
 width: 14px; 
height: 9px; 
background-color: #FF911C;
}
QDateTimeEdit::up-button:disabled {
 width: 14px; 
height: 9px; 
background-color: #CCCCCC;
}

QDateTimeEdit::up-button:hover {
 background-color: #FFD65E;
 border-top-right-radius: 5px; /* same radius as the QComboBox */
 
}
 
QDateTimeEdit::down-arrow:enabled { 
border-left: 3px solid none;
border-right: 3px solid none; 
border-top: 3px solid black; 
width: 0px; 
height: 0px; 
}

QDateTimeEdit::down-arrow:disabled { 
border-left: 3px solid none;
border-right: 3px solid none; 
border-top: 3px solid black; 
width: 0px; 
height: 0px; 
}


QDateTimeEdit::down-button:enabled { 
width: 14px; 
height: 9px;
 background-color: #FF911C;
}
QDateTimeEdit::down-button:disabled { 
width: 14px; 
height: 9px;
 background-color: #CCCCCC;
}
QDateTimeEdit::down-button:hover {
 background-color: #FFD65E;
 border-bottom-right-radius: 5px;
}

QDateTimeEdit::up-arrow:hover { 
border-left: 3px solid none;
border-right: 3px solid none;
 border-bottom: 3px solid black;
 width: 0px; 
height: 0px; 
}
QDateTimeEdit::down-arrow:hover {
 border-left: 3px solid none;
border-right: 3px solid none;
 border-top: 3px solid black;
 width: 0px;
 height: 0px; 
}</string>
        </property>
        <property name="locale">
         <locale language="Ukrainian" country="Ukraine"/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Час повідомлення:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTimeEdit" name="teNotification">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QTimeEdit:enabled {
 border: 1px solid #ff911c;
 border-radius: 5px;
 background-color: white;
 }
QTimeEdit:disabled {
 border: 1px solid #CCCCCC;
 border-radius: 5px;
 background-color: white;
 }

QTimeEdit::up-arrow:enabled { 
border-left: 3px solid none;
border-right: 3px solid none;
 border-bottom: 3px solid black; 
width: 0px;
 height: 0px; 
}
QTimeEdit::up-arrow:disabled { 
border-left: 3px solid none;
border-right: 3px solid none;
 border-bottom: 3px solid black; 
width: 0px;
 height: 0px; 
}


QTimeEdit::up-button:enabled {
 width: 14px; 
height: 9px; 
background-color: #FF911C;
}
QTimeEdit::up-button:disabled {
 width: 14px; 
height: 9px; 
background-color: #CCCCCC;
}

QTimeEdit::up-button:hover {
 background-color: #FFD65E;
 border-top-right-radius: 5px; /* same radius as the QComboBox */

}
 
QTimeEdit::down-arrow:enabled { 
border-left: 3px solid none;
border-right: 3px solid none; 
border-top: 3px solid black; 
width: 0px; 
height: 0px; 
}

QTimeEdit::down-arrow:disabled { 
border-left: 3px solid none;
border-right: 3px solid none; 
border-top: 3px solid black; 
width: 0px; 
height: 0px; 
}


QTimeEdit::down-button:enabled { 
width: 14px; 
height: 9px;
 background-color: #FF911C;
}
QTimeEdit::down-button:disabled { 
width: 14px; 
height: 9px;
 background-color: #CCCCCC;
}
QTimeEdit::down-button:hover {
 background-color: #FFD65E;
 border-bottom-right-radius: 5px;
}

QTimeEdit::up-arrow:hover { 
border-left: 3px solid none;
border-right: 3px solid none;
 border-bottom: 3px solid black;
 width: 0px; 
height: 0px; 
}
QTimeEdit::down-arrow:hover {
 border-left: 3px solid none;
border-right: 3px solid none;
 border-top: 3px solid black;
 width: 0px;
 height: 0px; 
}</string>
        </property>
        <property name="locale">
         <locale language="Ukrainian" country="Ukraine"/>
        </property>
        <property name="currentSection">
         <enum>QDateTimeEdit::HourSection</enum>
        </property>
        <property name="displayFormat">
         <string>h:mm AP</string>
        </property>
        <property name="calendarPopup">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Числовий пріорітет: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="weight_comboBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QComboBox QAbstractItemView {
    margin-top:1px;
    background:white;
    border: 1px solid #ff911c;
    box-shadow: transparent;
    padding: 4px 4px 4px 4px;
    selection-background-color:transparent;
    selection-color:#ff911c;
border-radius:5px;

}
QComboBox:disabled {
height:9px;
border: 1px solid #CCCCCC;
border-radius: 5px;
padding: 3px 3px 3px 3px;
}
QComboBox:enabled {
height:9px;
border: 1px solid #ff911c;
border-radius: 5px;
padding: 3px 3px 3px 3px;
}

QComboBox::down-arrow {
margin-top:3px;
image: url(&quot;:rc/downArrowIcon&quot;);
 }

QComboBox::drop-down:enabled{
background-color: #ff911c;
width: 14px;
}

QComboBox::drop-down:disabled{
background-color: #cccccc;
width: 14px;
}

QComboBox::drop-down:hover{
background-color: #FFD65E;
width: 14px;
    border-top-right-radius: 5px; 
    border-bottom-right-radius: 5px;
}</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="leLocation">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit:enabled{
border: 1px solid #ff911c;
 border-radius:10px;
 background-color: palette(base);
}

QLineEdit:disabled{
 border-radius:10px;
border: 1px solid #CCCCCC; 
background-color: palette(base);
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Місце проведення: </string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
